src/Sarif.Driver/Sdk/MultithreadedAnalyzeCommandBase.cs

```diff
@@ -824,12 +825,25 @@ internal string GetConfigurationFileName(string configurationFilePath, IFileSyst
                     : null;
             }
 
+            if (!File.Exists(configurationFilePath))
```

Comment:
This is preparatory content for publishing standard configurations in our tool output directory. I'd like to use this to do things like 'run the standard security rule set'.